README.md

```diff
@@ -71,3 +69,28 @@ $ docker-compose down
 ```
 
 Find alert_stream container names with `docker ps` and shut down with `docker stop [name]`.
+
+Notes
+-----
+
+Note well that currently the repo contents are copied into the Docker image on build, so any changes to the code require rebuilding the image if using Docker.
+
+Also note that consumers with the same group ID share a stream so that only one consumer in the group will receive a message (as in a queue). To run multiple consumers each consuming all messages, each consumer needs a different group ID.
+
+**On Docker**
+
+To just run a Python terminal in the Docker environment:
+
+```
+$ docker run -it alert_stream python
+```
+
+To collect postage stamp cutouts to your local machine, you can mount a local directory and give the Docker container write access with, e.g., the following command:
+
+```
+$ docker run -it --network=host -v $PWD/stamps:/home/alert_stream/stamps:rw alert_stream python bin/printAlertStream.py my-stream echo-group --stampDir stamps
+```
+
+**On Avro**
+
+Using the Avro serialization format to enforce a schema for the reader/writer is turned on by default but can be turned off by setting encode=False and decode=False in send/poll.
```

Comment:
I'm curious why this requires a rebuild.  It seems like you could add an --encode/--decode argument to each of the producers and consumers.  Is it just then you could potentially get things out of sync?